sympy/printing/pretty/pretty_symbology.py

```diff
@@ -564,13 +564,8 @@ def annotated(letter):
     information.
     """
     ucode_pics = {
-        'F': (2, 0, 2, 0, u('\N{BOX DRAWINGS LIGHT DOWN AND RIGHT}\N{BOX DRAWINGS LIGHT HORIZONTAL}\n'
-                            '\N{BOX DRAWINGS LIGHT VERTICAL AND RIGHT}\N{BOX DRAWINGS LIGHT HORIZONTAL}\n'
-                            '\N{BOX DRAWINGS LIGHT UP}')),
-        'G': (3, 0, 3, 1,
-              u('\N{BOX DRAWINGS LIGHT ARC DOWN AND RIGHT}\N{BOX DRAWINGS LIGHT HORIZONTAL}\N{BOX DRAWINGS LIGHT ARC DOWN AND LEFT}\n'
-                '\N{BOX DRAWINGS LIGHT VERTICAL}\N{BOX DRAWINGS LIGHT RIGHT}\N{BOX DRAWINGS LIGHT DOWN AND LEFT}\n'
-                '\N{BOX DRAWINGS LIGHT ARC UP AND RIGHT}\N{BOX DRAWINGS LIGHT HORIZONTAL}\N{BOX DRAWINGS LIGHT ARC UP AND LEFT}'))
+        'F': (2, 0, 2, 0, u'\u250c\u2500\n\u251c\u2500\n\u2575'),
+        'G': (3, 0, 3, 1, u'\u256d\u2500\u256e\n\u2502\u2576\u2510\n\u2570\u2500\u256f')
```

Comment:
I too agree that ``u`` should be called in a few special cases. I will point out the set of files which must have ``u`` (special cases) after going through the remaining ones:

As of now after the above conversation I think 

a.) ``sympy/printing/pretty/pretty_symbology.py``
b.) ``sympy/vector/dyadic.py ``
c.) ``sympy/vector/vector.py``
d.) ``sympy/vector/scaler.py``

Should be the ones which must use ``u`` from ``sympy.compatibility``.